src/app/components/team/Statuses/EditStatusDialog.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/sort-prop-types */
```

Comment:
@danielevalverde can you please also remove this directive and sort the prop types?

Reply:
Done :white_check_mark: